main.cpp

```diff
@@ -13,75 +13,272 @@ What you create in this project will be used as the basis of Project 5 in the co
-
-Main Object: Car Interior
-Sub Object 1: Steering Wheel
-    Name 4 things you'll find on the:    Steering Wheel
-        1) paddle shifters
-        2) 'cruise control' controls
-        3) 
-        4) 
-    Name 2 things you can do with the:   Steering Wheel
-        1) adjust cruise control settings.
-        2)
-        
-Sub Object 2: Instrument Cluster
-    Name 4 things you'll find on the:   Instrument Cluster
-        1)
-        2)
-        3)
-        4)
-    Name 3 things you can do with the:   Instrument Cluster
-        1)
-        2)
-        3)
+1) write the 10th UDT
+    come up with an object that is made of 5 smaller parts.
+    These parts will not be defined using Primitives, but instead will be their own UDTs you'll define in Part 1d
     
-Sub Object 3: Environment Controls
-    Name 3 things you'll find on the:    Environment Controls
-        1)
-        2)
-        3)
-    Name 3 things you can do with the:   Environment Controls
-        1)
-        2)
-        3)
-
-Sub Object 4: Infotainment System
-    Name 3 things you'll find on the:    Infotainment System
-        1)
-        2)
-        3)
-    Name 3 things you can do with the:   Infotainment System
-        1)
-        2)
-        3)
-
-Sub Object 5: Seat 
-    Name 3 things you'll find on the:    Seat
-        1)
-        2)
-        3)
-    Name 2 things you can do with the:   Seat
-        1)
-        2)
+    Here is an example:
+    Cell Phone
+
+    A Cell Phone is made up of the following 5 properties:
+        Display
+        Memory
+        CPU
+        Radio
+        Applications
+
+    A Cell Phone has 3 things it can do:
+        make a call
+        send a text
+        run an application.
+
+    Notice that I did not use "has a display" or "Has memory" or "has a cpu" as one of the properties of the CellPhone
+    
+    Writing 'has a ___" checks whether or not your object **has the ability to do something**.
+    Instead, I wrote "Display" or "CPU".  These are specific objects or amounts. 
+    
+    In C++ terms, this means to I want you to avoid using 'bool' (has a) as a member variable type.
+    Instead, prefer the other primitive types.
+
+    When you choose your 5 smaller parts, remember that each of these 5 Sub Objects will need 
+    to be defined with 5 primitive properties and 3 actions EACH.  
+
+2) write the name of the primitive type you'll be using after each property in UDTs 1-4:
+    pick properties that can be represented with 'int float double bool char std::string'
+    example: 
+        Display:
+            Number of Pixels (int)
+            Amount of Power consumed (milliwatt-hours) (float)
+            Brightness (double)
+            area in cm2 (int)
+            brand (std::string)
+
+3) Move the Thing 10 pseudo-code to after your Thing1-4 in the project.
 */
+
+/*
+Thing 10)
+5 properties:
+    1)
+    2)
+    3)
+    4)
+    5)
+3 things it can do:
+    1)
+    2)
+    3)
+ */
```

Comment:
You didn't follow instructions 1, 2 or 3. 
and now your codebase has 14 UDTs. 
- [ ] revise